packages/moonstone/Scrollable/Scrollable.js

```diff
@@ -602,18 +610,35 @@ class ScrollableBase extends Component {
 				this.uiRef.current.lastInputType = 'arrowKey';
 
 				direction = getDirection(keyCode);
-				if (overscrollEffectOn.arrowKey && !(element ? getTargetByDirectionFromElement(direction, element) : null)) {
-					const {horizontalScrollbarRef, verticalScrollbarRef} = this.uiRef.current;
+				if (overscrollEffectOn.arrowKey) {
+					if (!element) {
+						const {horizontalScrollbarRef, verticalScrollbarRef} = this.uiRef.current;
 
-					if (!(horizontalScrollbarRef.current && horizontalScrollbarRef.current.getContainerRef().current.contains(element)) &&
-						!(verticalScrollbarRef.current && verticalScrollbarRef.current.getContainerRef().current.contains(element))) {
-						this.checkAndApplyOverscrollEffectByDirection(direction);
+						if (!(horizontalScrollbarRef.current && horizontalScrollbarRef.current.getContainerRef().current.contains(element)) &&
+							!(verticalScrollbarRef.current && verticalScrollbarRef.current.getContainerRef().current.contains(element))) {
+							this.checkAndApplyOverscrollEffectByDirection(direction);
```

Comment:
This code will be called in different conditions from the original intention.

Reply:
Before is 
`if (overscrollEffectOn.arrowKey && !(element ? getTargetByDirectionFromElement(direction, element) : null)) `
After is
`if (overscrollEffectOn.arrowKey) && (!element) `
And, 
`if (overscrollEffectOn.arrowKey && element && getTargetByDirectionFromElement(direction, element) `
is covered on onNavNoTarget handler. 
So, I think it's same.

Did I miss something?

Reply:
Looks right.